### ISSUE TITLE
PR Environment Web build process flicker

### ISSUE THREAD

**Issue by howardbrande** [department-of-veterans-affairs/va.gov-cms#1016]
AC
* PRs don't fail for random reasons 

**Describe the defect**
Verfiy platform task failed: http://devshop.cms.va.gov/node/12876

```
[4/4] Building fresh packages...
error /var/aegir/projects/demo/vamcupgrade/docroot/vendor/va-gov/web/node_modules/puppeteer: Command failed.
Exit code: 1
Command: node install.js
Arguments: 
Directory: /var/aegir/projects/demo/vamcupgrade/docroot/vendor/va-gov/web/node_modules/puppeteer
Output:
ERROR: Failed to download Chromium r594312! Set "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" env variable to skip download.
{ Error: self signed certificate in certificate chain
    at TLSSocket.onConnectSecure (_tls_wrap.js:1051:34)
    at TLSSocket.emit (events.js:189:13)
    at TLSSocket._finishInit (_tls_wrap.js:633:8) code: 'SELF_SIGNED_CERT_IN_CHAIN' }
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Script cd web && yarn install handling the va:web:install event returned with error code 1
Script @va:web:install was called via post-install-cmd
```
It's just a flicker, the very next build was successful: http://devshop.cms.va.gov/node/12879
Note: We should ensure documentation covers this event and ensures those that are being trained of the possibility that this happens. Right now the CMS has no indicator of that happening

**Comment by ElijahLynn** [comment]
Thanks for reporting this here Howie. 

The `SELF_SIGNED_CERT_IN_CHAIN` is the type of thing that shouldn't go away, and should be consistent. I'll think on this for a bit. 

cc @jonpugh

**Comment by jonpugh** [comment]
Just saw it again:

**Comment by jonpugh** [comment]
This concerned me:

> ERROR: Failed to download Chromium r594312!

Are we downloading all of Chromium on every web build??

**Comment by ElijahLynn** [comment]
> This concerned me:
> 
> > ERROR: Failed to download Chromium r594312!
> 
> Are we downloading all of Chromium on every web build??

I think we are as the build script does this. We can set it to turn it off but don't really want to deviate from the build script defaults unless we are forced to.

**Comment by ElijahLynn** [comment]
Fixed in #1162.
